app/code/Magento/Developer/Console/Command/QueryLogEnableCommand.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Developer\Console\Command;
+
+use Magento\Framework\DB\Logger\LoggerProxy;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\Framework\App\DeploymentConfig\Writer;
+use Magento\Framework\Config\File\ConfigFilePool;
+
+class QueryLogEnableCommand extends Command
+{
+    /**
+     * input parameter log-all-queries
+     */
+    const INPUT_ARG_LOG_ALL_QUERIES = 'log-all-queries';
+
+    /**
+     * input parameter log-query-time
+     */
+    const INPUT_ARG_LOG_QUERY_TIME = 'log-query-time';
+
+    /**
+     * input parameter log-call-stack
+     */
+    const INPUT_ARG_LOG_CALL_STACK = 'log-call-stack';
+
+    /**
+     * command name
+     */
+    const COMMAND_NAME = 'dev:query-log:enable';
+
+    /**
+     * Success message
+     */
+    const SUCCESS_MESSAGE = "DB query logging enabled.";
+
+    /**
+     * @var Writer
+     */
+    private $deployConfigWriter;
+
+    /**
+     * QueryLogEnableCommand constructor.
+     * @param Writer $deployConfigWriter
+     * @param null $name
+     */
+    public function __construct(
+        Writer $deployConfigWriter,
+        $name = null
+    ) {
+        parent::__construct($name);
+        $this->deployConfigWriter = $deployConfigWriter;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName(self::COMMAND_NAME)
+            ->setDescription('Enable DB query logging');
+
+        $this->addArgument(
```

Comment:
please change arguments to options, so that it will allow named parameters vs positional.